Cargo.toml

```
@@ -18,12 +18,14 @@ crate-type = ["rlib"]
 wasm_bindgen = ["chrono/wasmbind", "js-sys"]
 
 [dependencies]
-ahash = { version = "0.8.12", features = ["serde"] }
-chrono = { version = "0.4.38", features = [
+browserslist-data = { path = "data" }
+
+ahash = { workspace = true, features = [ "serde" ] }
```

```suggestion
ahash = { workspace = true, features = [ "serde" ] }
browserslist-data = { path = "data" }
```

If `browserslist-data` will be published, version is also required here.